test/conformance/feature/conformance_feature.go

```diff
@@ -110,10 +110,10 @@ func conformanceFeature(featureName string, tls bool) *feature.Feature {
 	}
 
 	f.Setup("Start server", conformance_server.StartPod(server, port, tls))
-	f.Setup("Wait for server ready", func(ctx context.Context, t feature.T) {
+	f.Requirement("Wait for server ready", func(ctx context.Context, t feature.T) {
 		k8s.WaitForPodRunningOrFail(ctx, t, server)
 	})
-	f.Setup("Start client", func(ctx context.Context, t feature.T) {
+	f.Requirement("Start client", func(ctx context.Context, t feature.T) {
```

Comment:
This fails because "Start client" may run before "server ready" completes, I'm proposing this approach https://github.com/knative-sandbox/control-protocol/pull/246